src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { MAILER_API_URL } from '../config/config.example.json';
+
+export function uploadToS3(files) {
```

Comment:
1. add a comment section above this function like other files to describe param types instead
1. you should be returning an `A[iResponse` object like other functions in this directory, take a look at any file for examples.